.gitmodules

```diff
@@ -19,3 +19,6 @@
 [submodule "scipy/_lib/unuran"]
 	path = scipy/_lib/unuran
 	url = https://github.com/scipy/unuran.git
+[submodule "HiGHS"]
+	path = scipy/_lib/highs
+	url = https://github.com/mckib2/highs
```

Comment:
Can we move this to https://github.com/scipy/highs before merging?

Reply:
Yes - I had meant to give you permissions to it to move it over.  Let me do that real quick